orttraining/orttraining/python/training/orttrainer.py

```diff
@@ -14,6 +14,7 @@
 
 from onnxruntime.tools.symbolic_shape_infer import SymbolicShapeInference
 
+
```

Comment:
Nit: Can we exclude this file from the merge? The changes are unrelated.

Reply:
My editor auto formats opened files during save :P
Reverted